apps/defi/src/components/Topnav/components/Topnav.tsx

```diff
@@ -34,10 +42,15 @@ export const Topnav = () => {
   const isSm = useMediaQuery(theme.breakpoints.down('md'));
   const accountMenuAnchorEl = useRef(null);
   const [accountMenuOpen, setAccountMenuOpen] = useState(false);
+  const alertMenuAnchorEl = useRef(null);
+  const [alertMenuOpen, setAlertMenuOpen] = useState(false);
   const [drawerOpen, setDrawerOpen] = useState(false);
   const { status, pendingCount } = useActivitiesStatus();
+  const isOethRebaseRequired = useIsRebaseBannerVisible(tokens.mainnet.OETH);
```

Comment:
maybe better var names
`isNonRebasingOETH`
`isNonRebasingOUSD`

A rebase is a single action so the name used here seems a little off.